src/load_inputs/load_generators_data.jl

```diff
@@ -213,5 +213,280 @@ function load_generators_data!(setup::Dict, path::AbstractString, inputs_gen::Di
 			#   thus the overall is MTons/GW, and thus gen_in[g,:CO2_per_Start] is ton
 		end
 	end
+
+	load_vre_stor_data!(setup, path, inputs_gen, gen_in)
```

Comment:
1) I'm not sure why it's necessary to pass `gen_in` since that dataframe is already part of `input_gen`.
2) Generally the modified argument should come first. Here it's `inputs_gen`, not `setup` or `path`.